go/codeql-tools/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@rules_pkg//pkg:mappings.bzl", "pkg_attributes", "pkg_filegroup", "pkg_files")
+
+pkg_files(
+    name = "sh-files",
+    srcs = glob(["*.sh"]),
+    attributes = pkg_attributes(mode = "0755"),
```

Comment:
In theory, we could set this on the higher-level rule. The internal `dist.bzl` wrappers iirc set all files as executable - so there's no need to split these rules up here.

Reply:
while it's probably mostly harmless, I would like to avoid setting everything with the executable bit